.vscode/settings.json

```diff
@@ -1,4 +1,12 @@
 {
-  "typescript.tsdk": "./node_modules/typescript/lib",
-  "typescript.enablePromptUseWorkspaceTsdk": true
```

Comment:
I think we don't want to remove these lines

Reply:
2ae9a4a